pkg/exporter/syslogexporter/syslog_test.go

```diff
@@ -72,28 +56,3 @@ func TestFormatRFC5424(t *testing.T) {
 	assert.Equal(t, true, strings.Contains(formattedMsg, "UserID=\"Tester2\""))
 	assert.Equal(t, true, strings.Contains(formattedMsg, "PEN=\"27389\""))
 }
-
-func TestFormatRFC5424NoAddStructuredData(t *testing.T) {
```

Comment:
should we keep this test?

Reply:
I keep this test  but I merged tests which were in two functions. I left test cases which are relevant after deleting support for adding additional structured data.